lib/determine-basal/determine-basal.js

```diff
@@ -326,7 +326,14 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     }
 
     // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85
-    var threshold = min_bg - 0.5*(min_bg-40);
+
+    // But if profile.threshold_setting is over 65 increase the very minimum glucose threshold
+    var minimum_threshold = 65;
+    if (minimum_threshold) {
```

Comment:
Is this right? 
The `if` will always pass, and the `Math.min` expression will not have any effect (not assigned to anything).

Reply:
Thanks. I fixed it now